crawler/crawlutils.py

```diff
@@ -231,18 +230,18 @@ def snapshot_vms(
 
         metadata = {
             'namespace': namespace,
-            'features': features,
+            'features': ','.join(map(str, features)),
```

Comment:
This is needed because the tests are checking for this format `os,cpu` instead of this format: `['os','cpu']`.